src/openforms/logging/models.py

```diff
@@ -8,9 +9,9 @@
 
 from timeline_logger.models import TimelineLog
 
-from openforms.forms.models import Form
-from openforms.logging.constants import TimelineLogTags
-from openforms.submissions.models import Submission
```

Comment:
Why did we move this inside the methods?Does it have to do with the models and if they are ready or something else?

Reply:
yeah circular imports, I don't particularly like this :(

Reply:
I guess we cannot avoid it sometimes